SmartDeviceLink/SDLTemplateConfiguration.h

```diff
@@ -22,17 +22,18 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (instancetype)initWithPredefinedLayout:(SDLPredefinedLayout)predefinedLayout;
 
+#ifndef __cplusplus
```

Comment:
Can you add a description that says we are doing this because the library will not compile if imported into an Objective-C++ file because `template` is keyword in Objective-C++.

Reply:
Done